app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,74 @@
+class TweetController < ApplicationController
+
+    get '/tweets' do
+        if !!session[:id]
+            @user = User.find(session[:id])
```

Comment:
we can remove all of these `@user = User.find(session[:id])` since we have the `current_user` method now.